### DISCUSSION TITLE
Insiders release

### DISCUSSION THREAD

**Discussion by daiyam** [VSCodium/vscodium#1057] [Ideas]
Hi,

I'm planning to add Insiders release of VSCodium.

Here is a checklist of possible changes:

- use an `insider` branch
- use Actions and Releases of https://github.com/VSCodium/vscodium-insiders
- weekly release (might change in the future)
- update https://github.com/VSCodium/update-api to support `insider` quality
- update https://github.com/VSCodium/versions to differentiate `insider`/`stable` qualities
- update `update_version.sh` to differentiate `insider`/`stable` qualities

What do you think? Did I missed something?

**Comment by paulcarroty** [comment]
Distribution?

**Reply by daiyam** [reply]
As Releases in https://github.com/VSCodium/vscodium-insiders, no?

**Reply by paulcarroty** [reply]
Yeah. I can't host the Linux builds on Gitlab Pages due to size limit.

**Comment by gijoy** [comment]
Mr. daiyam can you please activate the insider build again...? I love using the appimage of insider build for linux... Thanks...

**Reply by gijoy** [reply]
Thank You Mr Daiyam...

**Reply by gijoy** [reply]
Mr Daiyam, help... Workflow insider-spearhead failed since 3 days ago...
